### PR TITLE
blkpr: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/blkpr.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`blkpr -c reserve -k {{clave_de_reserva}} {{-t|--type}} exclusive-access {{ruta/al/dispositivo}}`
+
+- Adelanta la reserva existente con una clave dada y la reemplaza por una nueva reserva:
+
+`blkpr -c preempt {{-K|--oldkey}} {{clave_antigua}} --key {{nueva_clave}} -t write-exclusive {{ruta/al/dispositivo}}`
+
+- Libera una reserva con una clave y [t]ipo dados en un dispositivo determinado:
+
+`blkpr -c release --key {{clave_reserva}} -t {{tipo_reserva}} {{ruta/al/dispositivo}}`
```
Comment (tricantivu) [inline pages.es/linux/blkpr.md:20]:
```suggestion
`blkpr -c release --key {{clave_de_reserva}} -t {{tipo_de_reserva}} {{ruta/al/dispositivo}}`
```

File: pages.es/linux/blkpr.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# blkpr
+
+> Registra, reserva, libera, anticipa y borra reservas persistentes en un dispositivo de bloque que admita reservas persistentes.
+> Más información: <https://manned.org/blkpr>.
+
+- Registra ([c]ommand) a una nueva reserva con una clave dada en un dispositivo determinado:
```
[the remark this answers is not in the document]
Comment (kant) [reply pages.es/linux/blkpr.md:6]: @tricantivu  Registra ([c]omando) en una nueva reserva con una clave dada en un dispositivo determinado:

[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/linux/blkpr.md:6]: Why use the preposition `en`?

[the remark this answers is not in the document]
Comment (kant) [reply pages.es/linux/blkpr.md:6]: Registra un ([c]omando) con una nueva reserva con una clave dada en un dispositivo determinado:

[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/linux/blkpr.md:6]: I think that is not what the original command description meant.

[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages.es/linux/blkpr.md:6]: > I think that is not what the original command description meant.
> 
> The mnemonic is to indicate that `register` is a command.
> 
> Is this correct @vitorhcl?

Exactly, @tricantivu. Sorry for the delay.

[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/linux/blkpr.md:6]: No problem.

[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/linux/blkpr.md:6]: Given vitor's reply, may I apply the initial suggestion @kant?

[the remark this answers is not in the document]
Comment (kant) [reply pages.es/linux/blkpr.md:6]: No problem, Sir

**Review by tricantivu — APPROVED** [review]

File: pages.es/linux/blkpr.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# blkpr
+
+> Registra, reserva, libera, anticipa y borra reservas persistentes en un dispositivo de bloque que soporte Reservas Persistentes.
```
[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/linux/blkpr.md:3]: We can double quote it the words in English, how's that?

[the remark this answers is not in the document]
Comment (kant) [reply pages.es/linux/blkpr.md]: @tricantivu Updated!

**Pull request merged**